src/init/starship.bash

```diff
@@ -70,10 +70,10 @@ else
     if [[ -z "$PROMPT_COMMAND" ]]; then
         PROMPT_COMMAND="starship_precmd"
     elif [[ "$PROMPT_COMMAND" != *"starship_precmd" ]]; then
-        PROMPT_COMMAND="$PROMPT_COMMAND;starship_precmd"
+        PROMPT_COMMAND="${PROMPT_COMMAND%;};starship_precmd"
```

Comment:
Ref [Posix parameter expansion](https://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html#tag_02_06_02)

> ${parameter%word}
    Remove Smallest Suffix Pattern. The word shall be expanded to produce a pattern. The parameter expansion shall then result in parameter, with the smallest portion of the suffix matched by the pattern deleted.

Reply:
Just put a comment into the source code (instead of the PR) and we'll be ready to rumble!